apps/site/navigation.json

```
@@ -113,6 +113,10 @@
         "branding": {
           "link": "/about/branding",
           "label": "components.navigation.about.links.branding"
+        },
+        "eol": {
+          "link": "/eol",
```

should we change this to 
```suggestion
          "link": "/about/eol",
```

---

and if so, I suspect all links to it would need to update or introduce a redirect

---

Hi @bmuenzenmeyer ,
I’ve updated the EOL link and redirect as follows:
- Updated "eol": `{ "link": "/about/eol" }`
- Added redirect:
```
{
  "source": "/:locale/eol",
  "destination": "/:locale/about/eol"
}
```

Now the path `/en/eol` properly redirects to → ` /en/about/eol`

Could you please have a look and let me know if this looks good, so I can push the code?

---

yes, i think that'd be the right approach. if you are touching the code. can you look for other `eol` links and update them to `/about/eol`? We should be internally consistent but the redirect is there for folks that might have hotlinked to it since released.

---

> can you look for other eol links and update them to /about/eol?

Great!